webcompat/helpers.py

```diff
@@ -626,3 +627,8 @@ def prepare_form(form_request):
         json_data = form_request.get_json()
         form_data.update(json_data)
     return form_data
+
+
+def is_json_array(json_data):
```

Comment:
I think we want to call this `is_json_object` for clarity. When I think an array, I think of something that resembles a list: `[1,2,3]`. 

Or, maybe I'm not considering something else?

Reply:
nope you are right. :) my bad.